documentation/tutorials/sensitivity/PEcAn_sensitivity_tutorial_v1.0.Rmd

```diff
@@ -99,48 +105,41 @@ runids
 ```{r,eval=FALSE}
-library(PEcAn.assim.batch)
-obs <- load.L2Ameriflux.cf("~/output/dbfiles/Ameriflux_site_0-772/US-NR1.2006.nc")
-names(obs)
-```
+File_Path<-"/fs/data3/tmccabe/AMF_US-NR1_BASE_HH_9-1.csv" #This is a path to where the observations are stored. 
+File_format<-PEcAn.DB::query.format.vars(bety = bety, format.id = 5000000002) #This matches the file with a premade "format" or a template that describes how the information in the file is organized
+site<-PEcAn.DB::query.site(site.id = 772,bety$con)  
 
-Change all bad values from -9999 to not a number (NA), and plot it to see if it make sense
-```{r,eval=FALSE}
-obs[obs==-9999]=NA
-plot(obs$NEE)
-```
+obs<-PEcAn.benchmark::load_data(data.path = File_Path, format= File_format, time.row = File_format$time.row,  site = site, start_year = 2004, end_year = 2004) 
 
-To get something we can compare to, we also need to only retain good NEE. At the least, a u* filter is required. Let’s apply a simple one
-```{r,eval=FALSE}
-niwotnee <- obs$NEE
-niwotustar <- obs$UST
-niwotnee[niwotustar<0.2]=NA
+names(obs) #Notice that there are several time columns ("$year", "$datetime", "$posix") for most plotting, "$posix" will be the best time axis
+
+annualNEEobs<-aggregate(obs$NEE, by= list(month(obs$posix)), simplify=TRUE, FUN =mean, na.rm= TRUE)
+
+plottable<-align_data(model.calc = arun, obvs.calc = obs, align_method = "match_timestep", var= "NEE")
+head(plottable)
 ```
 
 ####C. Finally, we can finally compare model to data
-In the earlier tutorial, you also compared NEE to the ensemble model run. Here we will do the same except for each sensitivity run. Recall that we had to convert units and make a scatter plot. 
+In the earlier tutorial, you also compared NEE to the ensemble model run. Here we will do the same except we include each sensitivity run. Recall that we made a scatter plot. 
 
-Here is what you did last time: We converted units, which we do with the *udunits* package. The observations are in umol m-2 s-1. The model output is read in kg C ha-1 yr-1. *udunits* can convert among many units but cannot do the conversion from grams carbon to moles carbon, so we do that separately, with the factor of 12 g C per mol.
+When we aligned the data, we got a dataframe with the variables we requested in a ```{r}NEE.m``` and a ```{r}NEE.o``` format. The ```{r}.o``` is for observations, and the ```{r}.m ``` is for model. The posix column allows for easy plotting along a timeseries.  
 ```{r,eval=FALSE}
-modnee <- ud.convert(arun$NEE,"kg ha-1 yr-1","ug m-2 s-1")/12.0
-plot(niwotnee,modnee)
+plot(plottable$NEE.m,plottable$NEE.o)
 abline(0,1,col="red")
 ```
 
 And remember the formula for RMSE:
 ```{r,eval=FALSE}
-sqrt(mean((niwotnee-modnee)^2,na.rm = TRUE))	
+sqrt(mean((plottable$NEE.o-plottable$NEE.m)^2,na.rm = TRUE))	
 ```
 
 *na.rm* makes sure we don’t include missing or screened values in either time series.
```

Comment:
Maybe. It's wordier than the first one, but I think that's the the two different author's style of tutorial.  I think in general it's good to err on the side of too much info in a tutorial than too little, because people can just (and will) skip around to what they find interesting/useful. 

Also the exploratory stuff in this tutorial  different than in the other.  If there's anything that is literally the same I can edit it down.